components/org.wso2.carbon.identity.application.authenticator.basicauth/src/main/java/org/wso2/carbon/identity/application/authenticator/basicauth/BasicAuthenticator.java

```diff
@@ -78,6 +78,9 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import static org.wso2.carbon.identity.application.authentication.framework.util.FrameworkConstants.PASSWORD;
```

Comment:
don't we have constants in the basic authenticator for these?

Reply:
Fixed with https://github.com/wso2-extensions/identity-local-auth-basicauth/pull/140/commits/7b768d19dc063f94605b7091da5693496ed374da